pkg/internalapis/cnsvolumeoperationrequest/cnsvolumeoperationrequest.go

```diff
@@ -242,6 +259,14 @@ func (or *operationRequestStore) StoreRequestDetails(
 	updatedInstance.Status.VolumeID = operationToStore.VolumeID
 	updatedInstance.Status.SnapshotID = operationToStore.SnapshotID
 	updatedInstance.Status.Capacity = operationToStore.Capacity
+	if operationToStore.QuotaDetails != nil {
```

Comment:
Do we need FSS check here and at all places ?

Reply:
Added the FSS check in Get and Store functions.